mdformat_myst/plugin.py

```diff
@@ -52,6 +111,36 @@ def update_mdit(mdit: MarkdownIt) -> None:
     mdit.add_render_rule("fence", render_fence_html)
     mdit.add_render_rule("code_block", render_fence_html)
 
+    for name in container_names:
+        container_plugin(mdit, name="{" + name + "}", marker=":")
+
+
+def container_renderer(
+    node: RenderTreeNode, context: RenderContext, *args, **kwargs
+) -> str:
+    children = node.children
+    paragraphs = []
+    if children:
+        # Look at the tokens forming the first paragraph and see if
+        # they form a YAML header. This could be stricter: there
+        # should be exactly three tokens: paragraph open, YAML
+        # header, paragraph end.
+        tokens = children[0].to_tokens()
+        if all(
+            token.type in {'paragraph_open', 'paragraph_close'} or
+                _YAML_HEADER_PATTERN.fullmatch(token.content)
+            for token in tokens
+        ):
+            paragraphs.append('\n'.join(token.content.strip()
+                                        for token in tokens
+                                        if token.content))
```

Comment:
Can you run `tox` locally? This looks like this should be formatted by black and part of the issue in CI